my-pern-app/backend/src/server.ts

```diff
@@ -44,10 +44,10 @@ app.put('/api/updateMovie/:movieId', async (req, res, next)=>{
         const {newTitle, newSummary, newImdbLink, newRating} = req.body
         const sql = `
         update "movies"
-        set "title" = $2
-        set "summary" = $3
-        set "imdbLink" = $5
-        set "rating" = $5
+        set "title" = $2,
+            "summary" = $3,
+            "imdbLink" = $4,
+            "rating" = $5
         where ("movieId" = $1)
         returning *
         `;
```

Comment:
On a put request (and also above on the post request) you want to return a _single_ object, not an array of objects.

Also, on a post you want to return status 201, while on a delete you want to return a 204 without a body.